public/app/plugins/panel/canvas/Connections.tsx

```diff
@@ -89,6 +90,15 @@ export class Connections {
   };
 
   handleMouseLeave = (event: React.MouseEvent | React.FocusEvent) => {
+    if (event.relatedTarget instanceof HTMLImageElement) {
+      const relatedTarget = event.relatedTarget;
+      assertInstanceOf(relatedTarget, HTMLImageElement);
+      // If mouse is leaving INTO the anchor image, don't remove div
+      if (relatedTarget.getAttribute('alt') === connectionAnchorAlt) {
+        return;
+      }
+    }
+
```

Comment:
Potentially could combine this if block into one check i.e.
```suggestion
  if (event.relatedTarget instanceof HTMLImageElement && event.relatedTarget.getAttribute('alt') === CONNECTION_ANCHOR_ALT) { return... }
```

Reply:
Only if we don't have to assert type to access getAttribute().